x-pack/plugins/observability/e2e/journeys/exploratory_view.ts

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { journey, step, before } from '@elastic/synthetics';
+import { loginToKibana, waitForLoadingToFinish } from '@kbn/synthetics-plugin/e2e/journeys/utils';
```

Comment:
Would be great if Exploratory View doesn't import `loginToKibana` and `waitForLoadingToFinish` from within synthetics plugin.  

If synthetics plugin imports these from /observability/... instead, that is better as synthetics plugin already imports a lot of shared stuff from **@kbn/observability-plugin**